client/my-sites/subscribers/components/subscriber-list-container/subscriber-list-container.tsx

```diff
@@ -1,4 +1,5 @@
-import { numberFormat, translate } from 'i18n-calypso';
+import formatNumber from '@automattic/components/src/number-formatters/lib/format-number';
```

Comment:
You could also export the `format-number` in the same PR, and then you can just `import { formatNumber } from '@automattic/components`

...although it isn't semantically really a _component_.